diff --git a/3rdparty/eigen/eigen.cmake b/3rdparty/eigen/eigen.cmake
--- a/3rdparty/eigen/eigen.cmake
+++ b/3rdparty/eigen/eigen.cmake
@@ -4,8 +4,9 @@ ExternalProject_Add(
     ext_eigen
     PREFIX eigen
     # Commit point: https://gitlab.com/libeigen/eigen/-/merge_requests/716
-    URL https://gitlab.com/libeigen/eigen/-/archive/da7909592376c893dabbc4b6453a8ffe46b1eb8e/eigen-da7909592376c893dabbc4b6453a8ffe46b1eb8e.tar.gz
-    URL_HASH SHA256=37f71e1d7c408e2cc29ef90dcda265e2de0ad5ed6249d7552f6c33897eafd674
+    URL https://gitlab.com/api/v4/projects/15462818/repository/archive.tar.gz?sha=da7909592376c893dabbc4b6453a8ffe46b1eb8e
+    URL_HASH SHA256=2b2afe0ad9f2148e41dedf4f562901606ccd2c47b98422a4e707d6bf8d1a4416
+    DOWNLOAD_NAME eigen-da7909592376c893dabbc4b6453a8ffe46b1eb8e.tar.gz
     DOWNLOAD_DIR "${OPEN3D_THIRD_PARTY_DOWNLOAD_DIR}/eigen"
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ""
