diff --git a/3rdparty/find_dependencies.cmake b/3rdparty/find_dependencies.cmake
--- a/3rdparty/find_dependencies.cmake
+++ b/3rdparty/find_dependencies.cmake
@@ -1786,7 +1786,6 @@ if(BUILD_CUDA_MODULE)
                     CUDA::cusparse
                     CUDA::cublas
                     CUDA::cublasLt
-                    CUDA::culibos
                 )
             endif()
         endif()
